EngineLayer/Proteomics/PeptideWithSetModifications.cs

```diff
@@ -124,7 +129,13 @@ public int NumMods
 
         public int MissedCleavages
         {
-            get { return modPep.MissedCleavages; }
+            get
+            {
+                if (modPep == null)
+                    return 0;
```

Comment:
"int?" is nullable :) https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/